website/docs/Threshold Decryption.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: Threshold Decryption.md
+sidebar_position: 16
+---
+
+Threshold Decryption (TDec) is a programmable access control service, in which decryption rights are typically based on the verified fulfillment of predefined conditions.
+Access conditions can be EVM-based (e.g. does the designated recipient own this NFT), RPC-driven (e.g. did the designated recipient commence active mining before this block) or time-based (e.g. has a preset period of time elapsed, after which the recipient's requests will be ignored).
+These conditions are also composable and can be combined in any logical sequence or decision tree.  
+
+TDec involves splitting the secret (a decryption key) into multiples shares and distributing those among authorized and collateralized node operators (stakers in the Threshold network).
+A minimum number – a threshold – of those operators holding the key shares must be online and actively participate in partial decryptions that can subsequently be combined by the requester to reconstruct the original plaintext data.
+
+We refer to the _threshold_ and _shares_ as `m` and `n` and the overall configuration as `m-of-n`.
+Ie. in a `3-of-5` scheme, 3 out of 5 node operators are required to reconstruct the original plaintext data.
+
+## Creating Threshold Decryption objects
+
+`nucypher-ts` provides a simple builder function, `generateTDecEntities`.
+This function takes several parameters that define the the policy (e.g. `m` and `n`), returns objects for encrypting/decrypting data, and stores the configuration for later use.
+
+
+Besides passing in `m` (threshold) and `n` (shares) as defined above, we must also specify the following: 
```

Comment:
There was some conversation on the python side of using `threshold` instead of `m`, and `shares` instead of `n`. Something to think about for the language in general and for the SDK.